### ISSUE TITLE
Highlight/tag brokered metadata documents

### ISSUE THREAD

**Issue by nuest** [o2r-project/o2r-platform#97]
The `.erc` directory contains several kinds of metadata documents, which we should hightlight or make accessible. I see several ways to do this:

- Add a "show metadata documents" filter to the file tree, which highlights selected files (or hides others)
- Show some icon next to the file names tagging the document as a medata file, ideally including the used standard, such as "METS", "PREMIS", "codemeta"
- Add direct links to the metadata documents from the "Metadata" tab
- A combination of the above

This would also be useful for demonstration purposes.
(Originally, I wanted to create an issue _add download button metadata documents_, but that's not really what this is about.)

@MarkusKonk @7048730 What's your take on this?

**Comment by MarkusKonk** [comment]
@nuest I would prefer the icons. You write "such as", how do we know which one was used?

**Comment by nuest** [comment]
From the different areas of the UI, the metadata - stuff best fits into the "Files" section on the left hand side.
The information which metadata format is in which file can be detected based on the file name in directory `.erc`.

@7048730 What do you think would be useful to see / be linked to here? I think brokering is a relevant feature that a user should be made aware of. Can you please help me to extend the table below, then we can get in touch with the UI-folks again.

## Information to show

| **filename** | icon | link(s)
| ------ | ------ | ------ |
| `metadata_zenodo.json` | ![image](http://about.zenodo.org/static/img/logos/zenodo-black-2500.png) | http://developers.zenodo.org/ |
| `metadata_datacite.xml` | ![image](https://assets.datacite.org/images/logo-big.png) | https://schema.datacite.org/ |
| `metadata_codemeta.json` | "codemeta" | https://doi.org/10.5063/schema/codemeta-2.0 |
| `metadata_mods.xml` | ![image](https://www.loc.gov/standards/mods/v3/images/mods.gif) | https://www.loc.gov/mods/v3 |
| `metadata_eudat_b2share.json` | ![image](https://www.eudat.eu/sites/default/files/logo-b2share.png) | https://www.eudat.eu/services/userdoc/b2share  |

If no icon is available, we could just put the string in a "[chip](https://material.angular.io/components/chips/overview)":

![image](https://user-images.githubusercontent.com/1325054/31658076-52e37bf4-b330-11e7-9685-63d485430895.png)


## How to integrate

We could provide a structured JSON document with the information table above, but I'd prefer to keep this in the UI, and to derive this information as much as possible from the existing metadata files instead of having multiple points of change. For example, the used datacite format can be quickly extracted from the actual file, right?

So we would have a `metadata_config.json` file that can be used to "augment" a file name in the file listing with the information from the table above.

_Alternatively_, we would auto-generate this information based on the `o2r-meta broker` output and provide it via the API. @7048730 Do you think that is worth the effort?

### Side note

_The "file tree augmentation" could also be used for other files_, see for example this screenshot from my IDE: 

![image](https://user-images.githubusercontent.com/1325054/31657458-b3bd1612-b32e-11e7-98b6-4f604e846075.png)

**Comment by 7048730** [comment]
First of all we should prioritise on zenodo, datacite and codemeta brokering, because it has several advantages. Datacite would be my favourite set because it is compact yet extensible and basically used by zenodo, too. Codemeta is important because it adresses software explicitly while Datacite was designed for all kinds of research data. Mods is specialised on long term archival and is xml; it occurs to me as peripheral for now. b2share allows to specify a community wide MD set which could be Datacite again for our use cases.

Schemas and links should go into the archival package information "package slip". 

Concerning the broker: I think, it is reasonable to make the user aware of MD brokering but I see no advantage in exposing the results via API. They are only relevant to shipping and will be displayed and included in that case anyway.

**Comment by nuest** [comment]
When is the package slip created?

**Comment by 7048730** [comment]
We have not yet discussed that. Late in the process of ERC creating, I'd say. Maybe with brokering?

**Comment by nuest** [comment]
Yes, I'd say create it with every brokering, as it is an index of the brokered metadata in a way.

Warn if it exists but simply override it.

**Comment by 7048730** [comment]
ok. task is here https://github.com/o2r-project/o2r-meta/issues/54

**Comment by nuest** [comment]
So the UI _could_ get the name and link/schema information from the package slip, but I guess we will not add the logo to the package slip. @7048730 If you have nothing to add, I would hand this over to @jansule and @MarkusKonk to discuss how to integrate the information in the ui.

@MarkusKonk I suggest you start creating the information you need in a local json document and when you have everything that you need and can display the metadata logos, then we will see how to provide the information from the server side.
